pkg/operator/credentialsrequest/status.go

```diff
@@ -263,6 +281,15 @@ func computeStatusConditions(
 		Status: configv1.ConditionTrue,
 		Type:   configv1.OperatorUpgradeable,
 	}
+	// If the operator is disabled, and we do not have a parent cred in kube-system, we are not
```

Comment:
Did you mean 'If the operator is not disabled' here?

Reply:
Another good catch. :)